### PR TITLE
[shopsys] do not mock Events in tests

### PR DESCRIPTION
**Pull request by RobinDvorak**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Event classes are final in Symfony 5
|New feature| No
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No
|Fixes issues| Allow upgrade to Symfony 5
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/framework/tests/Unit/Component/Router/Security/RouteCsrfProtectorTest.php [hunk 1]
```diff
@@ -18,35 +19,31 @@ public function testSubRequest(): void
...
+        $event = new ControllerEvent(
+            $this->createMock(HttpKernelInterface::class),
+            new ControllerProtected(),
+            new Request(),
+            HttpKernelInterface::SUB_REQUEST,
+        );
 
         $routeCsrfProtector = new RouteCsrfProtector($annotationReader, $tokenManagerMock);
-        $routeCsrfProtector->onKernelController($eventMock);
+        $routeCsrfProtector->onKernelController($event);
```
Comment (vitek-rostislav) [inline packages/framework/tests/Unit/Component/Router/Security/RouteCsrfProtectorTest.php:30]: there are no assertions in this test now :thinking: I was surprised there is nothing like `assertExceptionIsNotThrown` in PHPUnit :smile: I discussed this with @grossmannmartin and we agreed that we would vote for adding `assertTrue(true)` at the end of the test, even thought we are not happy about it :smile:

File: packages/framework/tests/Unit/Component/Router/Security/RouteCsrfProtectorTest.php [hunk 2]
```diff
@@ -18,35 +19,31 @@ public function testSubRequest(): void
...
+        $event = new ControllerEvent(
+            $this->createMock(HttpKernelInterface::class),
+            new ControllerNotProtected(),
+            new Request(),
+            HttpKernelInterface::MASTER_REQUEST,
+        );
 
         $routeCsrfProtector = new RouteCsrfProtector($annotationReader, $tokenManagerMock);
-        $routeCsrfProtector->onKernelController($eventMock);
+        $routeCsrfProtector->onKernelController($event);
```
Comment (vitek-rostislav) [inline packages/framework/tests/Unit/Component/Router/Security/RouteCsrfProtectorTest.php:46]: this is the same story as `testSubRequest`, so could you please assert true here as well? :slightly_smiling_face:

**Review by vitek-rostislav — CHANGES_REQUESTED** [review]
Hi, good job there, I like the outcome! I have added just two notes (it is actually one duplicated note :smile: )

File: packages/framework/tests/Unit/Component/Router/Security/RouteCsrfProtectorTest.php (hunk 1 above)
Comment (RobinDvorak) [reply packages/framework/tests/Unit/Component/Router/Security/RouteCsrfProtectorTest.php:30]: Fixed here: https://github.com/shopsys/shopsys/pull/2490/commits/2e480f6a69cef675227ae735ee86e90ae4461a27

File: packages/framework/tests/Unit/Component/Router/Security/RouteCsrfProtectorTest.php (hunk 2 above)
Comment (RobinDvorak) [reply packages/framework/tests/Unit/Component/Router/Security/RouteCsrfProtectorTest.php:49]: Fixed here: https://github.com/shopsys/shopsys/pull/2490/commits/2e480f6a69cef675227ae735ee86e90ae4461a27

**Review by vitek-rostislav — APPROVED** [review]

**Pull request merged**